app/scripts/lib/channels/notifier.js

```diff
@@ -18,9 +18,10 @@ define(function (require, exports, module) {
   var EVENTS = {
     COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
     DELETE: 'fxaccounts:delete',
+    DEVICE_NAME_CHANGE: 'fxaccounts:device_name_change',
     PROFILE_CHANGE: 'profile:change',
     SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'internal:signed_out'
+    SIGNED_OUT: 'fxaccounts:logout'
```

Comment:
Given the recent hiccup we had with accidentally sending webchannel events that the browser wasn't prepared for, I'd like to hear more about this change and what behaviour it will cause.  I assume it's to support disconnecting the device when it's disconnected in the web UI, but will it have any unintended side-effects if there's e.g. a disconnect between device-login and web-content-login state?